cmake/CMakeLists.txt

```diff
@@ -0,0 +1,160 @@
+project(usb-1.0)
+cmake_minimum_required(VERSION 3.10)
+set(CMAKE_C_USE_RESPONSE_FILE_FOR_INCLUDES   OFF)
+set(CMAKE_CXX_USE_RESPONSE_FILE_FOR_INCLUDES OFF)
```

Comment:
can you share a reference what is this for?
I've never needed to use those

Reply:
I vaguely remember needing this for mingw compilation.
For some reason (my guess is cmake bug) cmake tries to pass on @CMakeFiles/Include syntax to the msys compiler which it doesnt support.

I'm guessing the CMAKE_CXX_USE_RESPONSE_FILE_FOR_INCLUDES isnt required here (spillover from another unrelated set of changes)